src/main/resources/static/templates/CSRFVulnerability/LEVEL_3/CSRFVulnerability.js

```
@@ -0,0 +1,40 @@
+  document
+    .getElementById("resetPassBtn")
+    .addEventListener("click", function () {
+      let url = getUrlForVulnerabilityLevel();
+      doGetAjaxCall(
+        CallbackResponse,
+        url +
+          "?id=" +
+          "&pass=" +
+          "&newpass=" +
+          document.getElementById("newpass").value +
+          "&rtkparam=7877864849",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)